bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -227,14 +149,6 @@
 				<label>One Call API 5 Day Forecast</label>
 				<description>This is the weather forecast in five days from the one call API.</description>
 			</channel-group>
-			<channel-group id="forecastDay6" typeId="oneCallDaily">
```

Comment:
Maybe you can help me out here and explain when this change became needed?

Reply:
I am not sure _when_ it became needed, probably this was working never before in a correct way and no one has really noticed ... if you have 300 or 600 channels does not make that large difference in the UI, because it is unlikely I guess one scroll trhough the expected 300 ones and then notices there are even more unexpected ones.